test/test.js

```diff
@@ -18,6 +18,9 @@ const App = () => {
     >
       <h1>Hello World!</h1>
       <button className="btn" onClick=${e => setToggle(!toggle)}>Toggle</button>
+      <div className=${css`https://unpkg.com/tailwindcss@^1.0/dist/tailwind.min.css`}>
```

Comment:
Can we change this url to `https://unpkg.com/tailwindcss@1.1.4/dist/tailwind.min.css` (with a pinned version) so that we can ensure any future changes to the tailwind package doesn't break this "test".

Reply:
Thought I did - fixed now :)